src/main/java/io/github/sefiraat/slimetinker/items/componentmaterials/CMManager.java

```diff
@@ -142,22 +129,22 @@ public CMManager() {
             String id = entry.getKey();
 
             // Tools, armour and kits (referenced through dummy)
-            if (cm.isValidToolRod() & traitManager.isEnabled(id, IDStrings.ROD)) {
+            if (cm.isValidToolRod() && traitManager.isEnabled(id, IDStrings.ROD)) {
```

Comment:
how did I...

Reply:
I've no idea - luckily it just means a bit of extra overhead - and it meant I learned how to bypass short-circuiting :D!

Reply:
Also, I didn't see them during the review soooooooooo 🥇

Reply:
I made this change, but everything except bulky works fine.